### ISSUE TITLE
Margin, paddding and border missing direction-independent properties

### ISSUE THREAD

**Issue by Tmpod** [ScottPierce/kotlin-html#31]
As the title suggests, the lib is missing a way to control each border, margin or padding direction independently, which is very limiting.

**Comment by ScottPierce** [comment]
We have the following:

```Kotlin
            paddingLeft(10.px)
            paddingRight(10.px)
            paddingTop(10.px)
            paddingBottom(10.px)
            paddingX(10.px)
            paddingY(10.px)
            
            marginLeft(10.px)
            marginRight(10.px)
            marginTop(10.px)
            marginBottom(10.px)
            marginX(10.px)
            marginY(10.px)
            
            borderLeft(10.px, BorderStyle.SOLID, Color.BLACK)
            borderRight(10.px, BorderStyle.SOLID, Color.BLACK)
            borderTop(10.px, BorderStyle.SOLID, Color.BLACK)
            borderBottom(10.px, BorderStyle.SOLID, Color.BLACK)
```

padding and margin have been in for a while, and I added the border directions last night. The border versions are available in the most recent version: `0.7.11`

**Comment by Tmpod** [comment]
That's weird. I'd swear I couldn't find those methods. Maybe I was in an older version and didn't realise? Anyways, this can be closed :)
